### PR TITLE
Add new token and crowdsale. Crowdsale is fixed price and uncapped now

### PR DESCRIPTION
**Pull request by AugustoL**

### CODE REVIEW

File: contracts/LifToken.sol [hunk 1]
```diff
@@ -1,472 +1,17 @@
...
+pragma solidity ^0.4.13;
 
 import "zeppelin-solidity/contracts/ownership/Ownable.sol";
-import "zeppelin-solidity/contracts/payment/PullPayment.sol";
-import 'zeppelin-solidity/contracts/math/SafeMath.sol';
-import './LifInterface.sol';
-import './LifDAOInterface.sol';
+import "zeppelin-solidity/contracts/token/MintableToken.sol";
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+import "zeppelin-solidity/contracts/lifecycle/Pausable.sol";
```
Comment (kvbik) [inline contracts/LifToken.sol]: do we have to import the `SafeMath` and `Ownable` too? i don't see it used in the code.

Comment (AugustoL) [inline contracts/LifToken.sol]: seems that we dont have to.

**Comment by kvbik** [comment]
good job on friday!! what do we miss here to merge it to `master`?:)))

**Comment by nberger** [comment]
@kvbik I think not much! Things missing, from the top of my head:

- add presale payments
- transfer funds to foundation wallet as soon as minCap is reached and until market maker low cap is hit
- all the market maker stuff
- add gen test command for pausing the token (pausing crowdsale has a command already)
- add more state assertions in gen test
- add a gen test command for the crowdsale finalize op

But we can work on those in further PRs so yeah next step is to cleanup the commits here and merge :)

**Comment by kvbik** [comment]
100% agree, looking forward!

**Comment by nberger** [comment]
Build looks like if it's broken here, but it's not. Last build was https://travis-ci.org/windingtree/LifToken/builds/264279082 which is green, so I guess it's just that travis and github are not friends today :)

**Pull request merged**